eo-maven-plugin/src/test/java/org/eolang/maven/it/SnippetTestCase.java

```diff
@@ -301,18 +304,54 @@ private static String jdkExecutable(final String name) {
      * @return Classpath.
      */
     private static String classpath() {
-        return String.format(
-            ".%s%s",
-            File.pathSeparatorChar,
-            System.getProperty(
-                "runtime.jar",
-                Paths.get(System.getProperty("user.home")).resolve(
+        final String runtime;
+        final String env = "runtime.jar";
+        final String property = System.getProperty(env);
+        if (SnippetTestCase.isNotRealPath(property)) {
+            runtime = Paths.get(System.getProperty("user.home"))
+                .resolve(
                     String.format(
                         ".m2/repository/org/eolang/eo-runtime/%s/eo-runtime-%1$s.jar",
                         "1.0-SNAPSHOT"
                     )
-                ).toString()
-            )
+                ).toString();
+            Logger.warn(
+                SnippetTestCase.class,
+                "The path provided by '%s' environment variable is not a real path. The default path '%' will be used",
+                env,
+                runtime
+            );
+        } else {
+            runtime = property;
+        }
+        return String.format(
+            ".%s%s",
+            File.pathSeparatorChar,
+            runtime
         );
     }
+
+    /**
+     * Checks if the path is real path.
+     * @param path String path to check.
+     * @return True if path is real.
+     */
+    private static boolean isNotRealPath(final String path) {
```

Comment:
@volodya-lombrozo it's better to use "positive" conditional branches: rename it to `is RealPath`